src/docs/components/docs-topbar/docs-topbar.css

```
@@ -1,19 +1,34 @@
-.docs-topbar::part(container) {
-  margin: 0 var(--ld-sp-4);
+.docs-topbar {
   min-width: 20rem;
-  padding: 0 var(--ld-sp-24);
+
+  &::part(container) {
+    margin: 0 var(--ld-sp-4);
+    padding: 0 var(--ld-sp-24);
+  }
 }
 
 .docs-topbar__logo {
   display: block;
+  margin-left: calc(-1 * var(--ld-sp-6));
+}
+
+.docs-topbar__links {
+  margin-right: 0;
+  width: var(--docs-toc-width);
+  padding-left: calc(var(--ld-sp-24));
```

```suggestion
  padding-left: var(--ld-sp-24);
```

I also think this looks a little odd. Can you maybe make a little design review with Nic, first?